documentation/commands/git-new-pull-request.md

```diff
@@ -12,5 +12,5 @@ git new-pull-request
 
 #### DESCRIPTION
 
-Opens a browser to a new pull request for the current branch.
+Opens a browser window to the "new pull request" page for the current branch on the code hosting platform.
```

Comment:
At the moment this is unrelated to this PR. Should this be updated to mention parent branch somewhere?